### ISSUE TITLE
Offline mode

### ISSUE THREAD

**Issue by Greenheart** [29ki/29k#2580]
### Summary

Allow local sessions to be completed offline. This requires downloading and caching video and audio files as well as images.m and animations.

### Motivation

This would make it possible to use the app in locations without good internet connections, such as outside of big cities. Thus making this more accessible to more people.

**Comment by gewfy** [comment]
Yes this has been up for discussion, but not something we can priorities anytime soon. Actually on Android it already caches all videos and audio. So if you play everything once it should work offline. But this is invisible to the user and we don't have a feature to "download" all assets for a specific session yet.
We've also considered bundling all video and audio assets with the native app, but it would make it 2-500MB larger, so not a great solution.

We'll get back to this one over time. Thanks for the idea @Greenheart :)
